packages/database/src/migrations/20210714013524-DeleteDuplicateCountriesFromEntityTable-modifies-data.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const duplicateCountryEntityIds = [
+  "'60da867061f76a20cf01e23f'", // duplicate Fiji
+  "'60eb91df61f76a7f770790a3'", // duplicate Papua New Guinea
+  "'60949c6061f76a622c020231'", // duplicate Samoa
+];
+
+const realCountryEntityIds = [
+  "'5d51f501f013d6320f3cf633'", // real Fiji
+  "'5d3f8844f327a531bfd8ad77'", // real Papua New Guinea
+  "'5df1b88c61f76a485cd1ca09'", // real Samoa
+];
+
+const findMatchingCountries = (duplicateId, realId) => {
+  // checks if both are Fiji entities
+  if (duplicateId === "'60da867061f76a20cf01e23f'" && realId === "'5d51f501f013d6320f3cf633'") {
```

Comment:
Following the earlier suggestion, this would become

```suggestion
  if (duplicateId === duplicateEntityIdByCountry.FJ && realId === realEntityIdByCountry.FJ) {
```